backend/App.js

```diff
@@ -11,6 +11,7 @@ app.get('/', (req, res) => {
 });
 
 app.use('/kingdom', troops);
+app.use('/kingdom', buildings);
```

Comment:
```suggestion
app.use('/kingdom/buildins', buildings);
```
the way you wrote is working too, but I think it would be much more readable, if we used it like I suggested: /kingdom/buildings. If you agree, I'll add a new task to refactor the troops as well (add it in line 13, and remove it in the troops.js router file).

app.use('/kingdom', something) I think is not obvious.

Reply:
Yea, I prefer the way you suggest! I'll change my part.

Reply:
great, thanks! I added a new task for rewrite the troops as well:)